app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,7 @@
+.welcome-intro {
+  background-image: image-url("landscape-mountains-nature-sky.jpg");
+  min-height: 100%;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size, min-height